GeometrickFX/Shapes/ConcreteDomain/Rhombicosidodecahedron.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Shapes.AbstractDomain;
+
+namespace Shapes.ConcreteDomain
+{
+    public class Rhombicosidodecahedron : IShape
+    {
+        private Dodecahedron _decahedron;
+        public Rhombicosidodecahedron(Dodecahedron decahedron)
```

Comment:
This is somehow different than the other shapes as it needs another shape in order to exist